### PR TITLE
added rss_feed for theand9 gsoc blog 

### PR DESCRIPTION
**Pull request by theand9**

### CODE REVIEW

File: gsoc.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 gsoc2020:
   theand9:
-    rss_fed: "rss-url" # TODO (add blog with gsoc tag)
-    project: 'stingray'
+    rss_fed: "https://medium.com/feed/@andsbmp17"
+    project: 'Stingray'
```
Comment (dpshelio) [inline gsoc.yml]:
```suggestion
    project: 'stingray'
```

File: gsoc.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ gsoc2019:
     rss_feed: "https://aquatiko.github.io/series/gsoc/index.xml"
     project: 'JuliaAstro'
   jredondopizarro:
-    rss_feed: "https://medium.com/feed/@jrpg1996" 
+    rss_feed: "https://medium.com/feed/@jrpg1996"
```
Comment (dpshelio) [inline gsoc.yml:61]: @theand9 careful with what you editor changes automatically. It's not a good practice to commit changes that are not related to what you are suggesting to change.

**Review by dpshelio — APPROVED** [review]

File: gsoc.yml (hunk 2 above)
Comment (theand9) [reply gsoc.yml:61]: Yes sorry will make sure next time
